data/ideas.json

```
@@ -1,4 +1,10 @@
 [
+    {
+        "title": "Music sharing web app",
+        "description": "online audio distribution platform and music sharing website, that enables its users to upload, promote, and share audio, as well as a digital signal processor enabling listeners to stream audio",
+        "level": "intermediate",
+        "tags": "music, web-app"
```

The first tag is using as a unique key to match idea and project
```suggestion
        "tags": "music-sharing, music, web-app"
```